core/block/import/syncer/icon.go

```diff
@@ -2,17 +2,22 @@ package syncer
 
 import (
 	"fmt"
-	"os"
 	"strings"
 
+	"github.com/ipfs/go-cid"
+
 	"github.com/anyproto/anytype-heart/core/block"
 	"github.com/anyproto/anytype-heart/core/block/editor/basic"
 	"github.com/anyproto/anytype-heart/core/block/editor/smartblock"
 	"github.com/anyproto/anytype-heart/core/block/simple"
 	"github.com/anyproto/anytype-heart/core/session"
 	"github.com/anyproto/anytype-heart/pb"
+	"github.com/anyproto/anytype-heart/pkg/lib/logging"
+	oserror "github.com/anyproto/anytype-heart/util/os"
 )
 
+var log = logging.Logger("import")
```

Comment:
Maybe it is better to make more specific name to this logger, we have two logger with name **import** already

Reply:
I think it's better to use one name, because it makes search in graylog easier (imho)